charts/karma/Chart.yaml

```diff
@@ -1,11 +1,12 @@
-apiVersion: v1
+apiVersion: v2
+type: application
 description: A Helm chart for Karma - an UI for Prometheus Alertmanager
 name: karma
 home: https://github.com/prymitive/karma
 sources:
   - https://github.com/wiremind/wiremind-helm-charts/tree/main/charts/karma
   - https://github.com/prymitive/karma
-version: 1.8.3
+version: 2.0.0
```

Comment:
Hm, actually, one last thing, could you add a minimum kubernetes version here?

Reply:
Sure! Is `>= 1.19-0` ok?

Reply:
Yes!

Reply:
Done, I forgot to say.